libgeometry/src/geo_surface.c

```diff
@@ -196,6 +196,29 @@ void geo_surface_fprintf_irap_external_zcoord( const geo_surface_type * surface,
   geo_surface_fprintf_irap__( surface , filename , zcoord );
 }
 
+geo_surface_type  * geo_surface_alloc_new( int ny, double xinc, double yinc,
```

Comment:
OK - I see; well - nothing to do about the file format, but I still think we should make the method signature sane.